### PR TITLE
Updating DE-Start docs

### PR DESCRIPTION
**Pull request by carolinewoodhams**
Lots to be updated here:

- Index: landing page for detailed guides, now includes DE-Start section
- DE-Start index: links to 3 methods - availability, booking (both for car parking only) and cancellation
- Availability, booking and cancellation guides

A few things to note:

- I'm not sure that the parameters are correct, as we have direct encashment for this partner, so for some bookings we are taking payment details
- Because of this we have asked the partner to use a new "payment" endpoint - I've copied the URL across for the booking and cancel methods, but I haven't checked if these are working yet
- I have updated the examples, but I'm not convinced the parameters are correct, and therefore the examples. This is one to check through when I've got a clear head again.

Posting this as a WIP PR - to come back to later.

### CODE REVIEW

File: site/hxapi/de-start/bkg.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+### Method
+
+POST
+
+### Endpoint
+
+The endpoint to use is:
+
+```
+https://payment.holidayextras.co.uk/legacy/carpark/CarParkCode
```
Comment (jameshhx) [inline site/hxapi/de-start/bkg.md]: Is this correct? The hostname `payment.holidayextras.co.uk` does no appear to resolve.

File: site/hxapi/de-start/bkg.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+
+## Worked Examples
+
+Below are worked examples of both the request and response for booking car parking using the DE-Start method.
+
+### Car Park Booking Request - DE-Start method
+
+```html
+https://payment.holidayextras.co.uk/legacy/carpark/FRA8
```
Comment (jameshhx) [inline site/hxapi/de-start/bkg.md]: Same as above - is this a valid URL? Domain doesn't resolve

File: site/hxapi/de-start/cancel.md [hunk 3]
```diff
@@ -2,108 +2,153 @@
...
-Post
+GET / POST
 
+### Endpoint
 
+The endpoint to use is (where `YourBookingRef` is the Holiday Extras booking reference):
 
-### Different requests
+```
+https://payment.holidayextras.co.uk/legacy/booking/YourBookingRef
```
Comment (jameshhx) [inline site/hxapi/de-start/cancel.md]: Same comment regarding hostname

File: site/hxapi/de-start/cancel.md [hunk 4]
```diff
@@ -2,108 +2,153 @@
...
+As of 12th January 2018 we no longer charge credit card fees, in line with the Payment Services Directive (PSD2). This applies to both UK and EU products.
+
+## Worked examples
+
+Below are worked examples of both the request and response for cancelling a booking.
+
+### Cancel Booking Request - Step 1
+
+```html
+https://payment.holidayextras.co.uk/legacy/booking/YourBookingRef?BusinessUnit=YourBusinessUnit&ConfirmCancel=N&Operator=BJT&SStartCustomer=YourStartCustomer&key=YourKey&token=YourToken&CancelRef=YourCancelRef
```
Comment (jameshhx) [inline site/hxapi/de-start/cancel.md]: And here - hostname

File: site/hxapi/de-start/cancel.md [hunk 5]
```diff
@@ -2,108 +2,153 @@
...
     </Request>
   </API_Header>
 </API_Reply>
 ```
 
-Confirmed cancel
+### Cancel Booking Request - Step 2
+
+```html
+https://payment.holidayextras.co.uk/legacy/booking/YourBookingRef
```
Comment (jameshhx) [inline site/hxapi/de-start/cancel.md]: Hostname

**Comment by carolinewoodhams** [comment]
Agree James - I will check that with Payment Pod (as well as the bulk of the docs with Anja). Thanks for the review!

**Pull request merged**